src/mito2/src/cache/file_cache.rs

```diff
@@ -37,8 +37,10 @@ use crate::sst::file::FileId;
 use crate::sst::parquet::helper::fetch_byte_ranges;
 use crate::sst::parquet::metadata::MetadataLoader;
 
-/// Subdirectory of cached files.
-const FILE_DIR: &str = "files/";
+/// Subdirectory of cached files for write.
+///
+/// This must contain three layers, corresponding to [`build_prometheus_metrics_layer`](object_store::layers::build_prometheus_metrics_layer).
```

Comment:
Just want to adapt to the new layer, and then we break changing the default cache directories. It's not a good idea totally.

Reply:
Despite the layer, this change also simplifies how users config the system

Reply:
Sure. You can retain the default paths as they were; they are separate matters. Simply the system config but keeps the default values.

Reply:
The previous one is not ideal. They are `greptimedb/object_cache/write/files/` and `object_cache/read/`

Reply:
Got it. My question is, why do we add the `greptimedb/` prefix to the `object_cache/{ read | write}`, it's ugly and unnecessary. If the metric layer needs the three layers of the path, I think we must change it.

Reply:
It's now changed to `cache/object/<read|write>`